README.md

```diff
@@ -69,12 +69,13 @@ For a complete list of SMM features please check out the [SMM docs](https://smm-
 ###  Build and deploy
 Download or check out the latest stable release.
 
-Run `make deploy` to deploy the operator controller-manager on your kubernetes cluster.
+Run `make deploy` to deploy the operator controller-manager on your kubernetes cluster.  To use the pre-built images do not set the TAG environment variable.
 
 Check if the controller is running in the `istio-system` namespace:
 ```
 $ kubectl get pod -n istio-system
-
+```
+```
```

Comment:
I think this was deliberately one section. First the command and then the output of the command in one section.